packages/v4/src/content/design-guidelines/content/units-and-symbols.md

```diff
@@ -0,0 +1,102 @@
+---
+id: Units and symbols
+section: UX writing
+---
+
+This page focuses on how to write units and symbols in an interface. For guidance on writing and displaying numeric values like date, time, and currency, see our [Numerics](/ux-writing/numerics) resource.
+
+Use consistent formatting, terminology, and symbols to display units of measure in your UI. Units of measure and symbols often appear when describing quantitative values including:
+
+- Date and time
+- Data storage
+- Data bandwidth
+- Currency
```

Comment:
Let's add periods to these line items since they're creating complete sentences with the intro clause.